### ISSUE TITLE
[Adaptive Cards-ExpenseReport.json]: At 200% resize settings, '$100' and '$4.30' dollar value text present for the '06/19: Auto Mobile Expense' and '06/21: Excess Baggage Cost' button control is getting cropped and not properly visible on the application.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9221]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0

**Pre-requisites:**
1. Go to system Settings-> System-> Display-> Scale & Layout-> Change the size of text, apps, and other items at 100%-> Display Resolution (1680*1050)
2. Go to system Settings-> Accessibility-> Text size-> 200

**Repro steps:**
1. Open the Adaptive Cards Visualizer application.
2. Navigate to the 'ExpenseReport.json' control present on left navigation using tab key and activate it using enter key.
3. 'ExpenseReport.json' adaptive card will appear on right sight of the pane.
4. Now navigate to the '$100' and '$4.30' dollar value text present for the '06/19: Auto Mobile Expense' and '06/21: Excess Baggage Cost' button control using tab key.
5. Verify that, at 200% resize settings, '$100' and '$4.30' dollar value text present for the '06/19: Auto Mobile Expense' and '06/21: Excess Baggage Cost' button control present on the 'ExpenseReport.json' card is properly visible on the application or not.

**Actual Result:**
 At 200% resize settings, '$100' and '$4.30' dollar value text present for the '06/19: Auto Mobile Expense' and '06/21: Excess Baggage Cost' button control present on the 'ExpenseReport.json' card is getting cropped and not properly visible on the application.

**Expected Result:**
 At 200% resize settings, '$100' and '$4.30' dollar value text present for the '06/19: Auto Mobile Expense' and '06/21: Excess Baggage Cost' button control present on the 'ExpenseReport.json' card should be properly visible without getting crop/overlap on the application.

**User Impact:**
Low vision users depend on the resize settings will be impacted if at 200% resize settings, '$100' and '$4.30' dollar value text present for the '06/19: Auto Mobile Expense' and '06/21: Excess Baggage Cost' button control present on the 'ExpenseReport.json' card is getting cropped and not properly visible on the application due to which users may be unable to know the complete information of the dollar value and might get confused on missing some informative information which lead to confusion, incorrect inputs, and a frustrating user experience.

**WCAG Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/1fb20e74-7131-45e9-85ff-c3aa7dcf8dfb)

### Card JSON

```JSON
N/A
```
